src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -360,22 +457,109 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
     });
 
     var coordinate = new GeoCoordinate().copy(coordCarto, UNIT.DEGREE);
+    // TODO: use GeoCoordinate conversion instead
+    var positionCamera = ellipsoid.cartographicToCartesian(coordinate);
+    this.scene = Scene(positionCamera, ellipsoid.size, viewerDiv, debugMode, gLDebug);
 
-    this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
+    this.initProviders(this.scene);
 
-    var map = new Globe(ellipsoid, gLDebug);
+    this.sceneLoadedDeferred = defer();
+    this.addEventListener('globe-loaded', () => {
+        this.sceneLoadedDeferred.resolve();
+        this.sceneLoadedDeferred = defer();
+    });
 
-    this.scene.add(map);
+    // init globe layer with default parameter
+    const wgs84TileLayer = {
+        protocol: 'tile',
+        id: globeLayerId,
+        update: (context, layer, node) => updateTreeLayer(processTiledGeometryNode, context, layer, node),
+        nodeType: TileMesh,
+        maxLevel: 16,
+        ellipsoid,
+        initRootNodes: initTiledGeometryLayer,
+        schemeTile: globeSchemeTileWMTS(globeSchemeTile1),
+        cullingTest: globeCulling,
+        preUpdate: (context, layer) => preGlobeUpdate(context, layer),
+        initNewNode,
+        mustSubdivide: globeSubdivisionControl,
```

Comment:
Because currently `processTiledGeometryNode` is slightly more complicated that simply chaining 3 functions.